Gruntfile.js

```diff
@@ -94,10 +102,8 @@ module.exports = function( grunt ) {
 			paths = lsOutput.trim().split( /\n/ ).filter( function( file ) {
 				return ! /^(blocks|\.|bin|([^/]+)+\.(md|json|xml)|Gruntfile\.js|tests|wp-assets|dev-lib|readme\.md|composer\..*|patches|webpack.*)/.test( file );
 			} );
-			paths.push( 'vendor/autoload.php' );
+			paths.push( 'vendor/**' );
```

Comment:
I noticed this is resulting in `vendor/cweagans/composer-patches` being included unexpectedly. Also, there are lot of `php-css-parser` files being included which should not be part of the build. In particular, `tests`. Only the files in `vendor/sabberworm/php-css-parser/lib/*` should be included.